app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsViewModelTest.kt

```diff
@@ -323,6 +352,47 @@ class ImportLoginsViewModelTest : BaseViewModelTest() {
             }
         }
 
+    @Test
+    fun `SuccessfulSyncAcknowledged should hide bottom sheet and send NavigateBack`() = runTest {
+        val viewModel = createViewModel()
+        turbineScope {
```

Comment:
You should check [this thing](https://github.com/bitwarden/android/blob/main/app/src/test/java/com/x8bit/bitwarden/ui/platform/base/BaseViewModelTest.kt#L14) out

Reply:
ah nice I had not seen this used but had seen turbine scope used directly, can change this with the other stuffs